src/commcare_cloud/ansible/deploy_commcarehq.yml

```diff
@@ -110,11 +110,14 @@
           - "{{ mgmt_cmd_managed_files|default([]) }}"
           - "{{ prometheus_managed_files|default([]) }}"
           - "{{ prometheus_django_runner_managed_files|default([]) }}"
-    - shell: find {{ service_home|quote }} -type f
+    - name: "Find supervisor config files"
+      become: true
```

Comment:
The commit message says "Run some tasks as cchq..." but this appears to be running as root. Is `become_user: '{{ cchq_user }}'` specified somewhere that I missed?

The same applies to the next task as well.

Reply:
Ah, yes. This was a git commit cleanup/interactive rebase error. I originally did do it as `cchq`, marco pointed out that it wasn't necessary to run as cchq specifically, and so I made a change and melded it into the same commit. The code ended up how I intended it, but the commit message ended up not correctly describing it.